src/modules/critics/ac.py

```diff
@@ -16,14 +16,33 @@ def __init__(self, scheme, args):
 
         # Set up network layers
         self.fc1 = nn.Linear(input_shape, args.hidden_dim)
-        self.fc2 = nn.Linear(args.hidden_dim, args.hidden_dim)
```

Comment:
Did you try training agents with IPPO and MAPPO with RNN critics and with default MLP critics? Does everything train nicely? Do you see that with MLP critics you get the same curves as before? We should make sure this is all solid before we merge this into the main repo.